WatchUI/robotframework.py

```diff
@@ -18,6 +18,9 @@
 import csv
 import pandas as pd
 
+SELENIUM_LIB = BuiltIn().get_library_instance("SeleniumLibrary")
+ROBOT_LIB = BuiltIn().get_library_instance("BuiltIn")
+
```

Comment:
Za mě by to mělo být malým písmem, protože volám metodu třídy. PEP8 mi hlásí že to je ok :-O